packages/graphql/tests/integration/types/localtime.int.test.ts

```diff
@@ -166,7 +166,7 @@ describe("LocalTime", () => {
 
             const mutation = /* GraphQL */ `
                     mutation ($id: ID!, $time: LocalTime) {
-                        ${Movie.operations.update}(where: { id_EQ: $id }, update: { time_SET: $time }) {
+                        ${Movie.operations.update}(where: { id_EQ: $id }, update: { time: {set: $time} }) {
```

Comment:
same reason as above:
```suggestion
                        ${Movie.operations.update}(where: { id: { equals: $id } }, update: { time: {set: $time} }) {
```